### DISCUSSION TITLE
Link webinar videos etc

### DISCUSSION THREAD

**Discussion by jdsika** [openMSL/.github#24] [Ideas]
We are linking relevant papers and code. We should link a collection of technical videos explaining parts of OpenMSL? @openMSL/sl1-maintainer

**Comment by jdsika** [comment]
https://youtu.be/EctmfTfW45Y?si=bZxY5OPCfhT2dvjd

**Comment by jdsika** [comment]
https://www.linkedin.com/posts/nicco-hagedorn_opendrive-visualization-with-open-source-activity-7112103461147549697-hTxa?utm_source=share&utm_medium=member_desktop

**Comment by 9h03n1x** [comment]
Hi everyone, if you have feedback and/or Video Ideas, please let me know.

**Comment by ClemensLinnhoff** [comment]
I added your suggested webinars to the documentation page.
Please check out the PR https://github.com/openMSL/doc/pull/4 and feel free to suggest additional webinars.
We should continue the discussion in the PR or after the merge in a new issue in the doc repository. Therefore, I will close this discussion.
